### PR TITLE
Impl

### PR DESCRIPTION
**Pull request by ganmacs**
ref: https://github.com/fluent/fluent-plugin-prometheus/pull/134

### CODE REVIEW

File: spec/fluent/plugin/out_prometheus_pushgateway_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'fluent/plugin/out_prometheus_pushgateway'
+require 'fluent/test/driver/output'
+
+RSpec.describe Fluent::Plugin::PrometheusPushgatewayOutput do
+  describe '#configure' do
+    describe 'valid' do
+      it 'creates successfully' do
+        config = %[
+          type prometheus_pushgateway
```
Comment (repeatedly) [inline spec/fluent/plugin/out_prometheus_pushgateway_spec.rb]: Use `@type`

File: spec/fluent/plugin/out_prometheus_pushgateway_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+          job_name name
+        ]
+        driver = Fluent::Test::Driver::Output.new(described_class)
+        expect { driver.configure(config) }.not_to raise_error
+      end
+
+      it 'raises an error when job name does not exist' do
+        config = 'type prometheus_pushgateway'
```
Comment (repeatedly) [inline spec/fluent/plugin/out_prometheus_pushgateway_spec.rb]: ditto

**Comment by repeatedly** [comment]
plugin itself is good.

**Comment by repeatedly** [comment]
client dependency should forward to fluent-plugin-prometheus?

File: spec/fluent/plugin/out_prometheus_pushgateway_spec.rb (hunk 1 above)
Comment (ganmacs) [reply spec/fluent/plugin/out_prometheus_pushgateway_spec.rb]: ah… good catch :bow: https://github.com/fluent/fluent-plugin-prometheus_pushgateway/pull/1/commits/fde0365da1efcd97b9ba1d966d1e04cc9d354783

**Comment by ganmacs** [comment]
I like to add it explicitly. but either is fine. I removed it https://github.com/fluent/fluent-plugin-prometheus_pushgateway/pull/1/commits/d3c97d7bc6d4c8c20eb5543ee7fc3beabda3bf44

**Comment by ganmacs** [comment]
https://github.com/fluent/fluent-plugin-prometheus/pull/134#discussion_r378652908

@cosmo0920 added! https://github.com/fluent/fluent-plugin-prometheus_pushgateway/pull/1/commits/9177dcb7b4834cf4c2cd578da2502c35eddd0965

**Comment by ganmacs** [comment]
Do you have any concerns?  @cosmo0920 @repeatedly

**Comment by ganmacs** [comment]
Thank you!

**Pull request merged**